src/functions-and-arrays.js

```diff
@@ -1,41 +1,96 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2) { return num1 }
+  else { return num2 }
+}
+result = maxOfTwoNumbers(3, 5);
```

Comment:
You don't need to call your function here to make it work. If you use it, to test your code, that's fine! 😉 But you should better remove these function calls before you submit the lab.